### DISCUSSION TITLE
hello

### DISCUSSION THREAD

**Discussion by PopcornEdward** [community/community#53411] [Actions]
### Select Topic Area

Question

### Body

first write

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @PopcornEdward  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be closed to keep our community nice and tidy 🧹.
